src/components/UI/organisms/AddMealForm/AddMealForm.css

```diff
@@ -0,0 +1,8 @@
+.AddMealForm {
```

Comment:
By the way, could you tell me a little bit more about the idea behind organising files? I mean the `UI/organisms` part. I'm asking because I haven't encountered this before.

Reply:
It is the so-called atomic design methodology. 

https://atomicdesign.bradfrost.com/chapter-2/#:~:text=Organisms%20are%20relatively%20complex%20UI,distinct%20sections%20of%20an%20interface.

MUI creators suggested it in the previous versions (don't know why). But in fact, due to making the code less readable in the end rather than making it easier to read, more often used is `modularization` (if the app is big) and `DDD` on the front-end like for example:

https://css-tricks.com/domain-driven-design-with-react/

Reply:
Thanks for the clarification!